pow/core/build.gradle

```diff
@@ -4,4 +4,9 @@ dependencies {
   implementation project(':ssz')
 
   implementation 'io.projectreactor:reactor-core'
+  implementation 'com.google.guava:guava'
+
+  testImplementation 'org.mockito:mockito-core'
+
+  testImplementation project(':core').sourceSets.test.output
```

Comment:
Is it some kind of hack?
If yes, I'd add comment

Reply:
Yep, it's naturally a hack.